testing/test-app/tests/rendering/aria-grid/node-selectors-test.ts

```diff
@@ -39,14 +39,16 @@ module('Rendering | node-selectors', function (hooks) {
   module('siblingsOf', function () {
     test('does not select nested cells in a nested grid', async function (assert) {
       await renderNestedGrid();
-
+      type Cell = {
+        id: string;
+      };
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       let firstCell = find('#cell-a')!;
       let siblings = siblingsOf(firstCell);
 
       assert.equal(siblings?.length, 2);
       assert.deepEqual(
-        siblings?.map((cell) => cell.id),
+        siblings?.map((cell: Cell) => cell.id),
```

Comment:
I was curious what the typescript error was here so I checked out `main` to have a look...

I did get an error about `any` but I *also* had an error on the `import` of `siblingsOf`.

I'm not too familiar with this project (especially since it converted to a v2 addon) but I wondered if this was a symptom of it needing to be *built* so the types could be found.

The root `package.json` references a `build` script so I ran that and then the type error here fixed itself...

So I think what we need to do is update the docs to make sure people remember to do that. I'm not sure to what extent we could automate it... #6 installed `turbo` but I don't see it used to e.g. "build package before trying to run tests"...

Reply:
Let me roll back that commit for the TS fix.

Reply:
oh! I discovered that by fixing the concurrently error, running `pnpm dev` probably fixes this, will update.